Automattic/woocommerce-payments
# WooPay enablement converts legacy UPE to split UPE

Fixes #5612 

#### Changes proposed in this Pull Request


# Enabling WooPay automatically enables split UPE for legacy UPE users


When WooPay is enabled from the WCPay settings. If the legacy UPE flag (`_wcpay_feature_upe`) is switched on, we must toggle this flag off, and toggle the split UPE flag (`_wcpay_feature_upe_split`) on instead.

WIP (maybe?).





#### Testing instructions


1. Ensure legacy UPE is enabled on site.
2. Enable WooPay from WCPay settings.
3. Legacy UPE should be automatically disabled and split UPE should be enabled instead.


-------------------


**Post merge**
